vl-convert-rs/src/converter.rs

```diff
@@ -330,9 +330,20 @@ function vegaToSvg(vgSpec, allowedBaseUrls, formatLocale, timeFormatLocale, erro
         vega.timeFormatLocale(timeFormatLocale);
     }
     let view = vegaToView(vgSpec, allowedBaseUrls, errors);
-    let svgPromise = view.toSVG().finally(() => {
-        view.finalize();
-        vega.resetDefaultLocale();
+    let svgPromise = view.runAsync().then(() => {
+        try {
+            // Workaround for https://github.com/vega/vega/issues/3481
+            view.signal("geo_interval_init_tick", {});
+        } catch (e) {
+            // No geo_interval_init_tick signal
+        }
+    }).then(() => {
+        return view.runAsync().then(
+            () => view.toSVG()
+        ).finally(() => {
+            view.finalize();
+            vega.resetDefaultLocale();
+        })
```

Comment:
It was necessary to call `runAsync`, then set `geo_interval_init_tick`, then call `runAsync` again